hr_holidays_leave_auto_approve/views/hr_holidays_status.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_holiday_status_form_auto_approve" model="ir.ui.view">
+        <field name="model">hr.leave.type</field>
+        <field name="inherit_id" ref="hr_holidays.edit_holiday_status_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='validation']">
+                <field name="auto_approve"/>
+            </xpath>
+        </field>
+    </record>
+
+</odoo>
```

Comment:
Could you rename this file, from `hr_holidays_status.xml ` to `hr_leave_type.xml`?